sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1391,30 +1397,22 @@ def _is_associative(func):
       and func.__module__ in ('numpy', 'builtins'))
 
 
+
+@populate_not_implemented(pd.core.groupby.generic.DataFrameGroupBy)
 class _DeferredGroupByCols(frame_base.DeferredFrame):
   # It's not clear that all of these make sense in Pandas either...
   agg = aggregate = frame_base._elementwise_method('agg')
   any = frame_base._elementwise_method('any')
   all = frame_base._elementwise_method('all')
-  apply = frame_base.not_implemented_method('apply')
-  backfill = bfill = frame_base.not_implemented_method('backfill')
   boxplot = frame_base.wont_implement_method('plot')
-  corr = frame_base.not_implemented_method('corr')
-  corrwith = frame_base.not_implemented_method('corrwith')
-  cov = frame_base.not_implemented_method('cov')
-  cumcount = cummax = cummin = cumprod = cumsum = (
-      frame_base.not_implemented_method('cum*'))
   describe = frame_base.wont_implement_method('describe')
   diff = frame_base._elementwise_method('diff')
-  dtypes = frame_base.not_implemented_method('dtypes')
-  expanding = frame_base.not_implemented_method('expanding')
-  ffill = frame_base.not_implemented_method('ffill')
   fillna = frame_base._elementwise_method('fillna')
   filter = frame_base._elementwise_method('filter')
   first = frame_base.wont_implement_method('order sensitive')
   get_group = frame_base._elementwise_method('group')
   head = frame_base.wont_implement_method('order sensitive')
-  hist = frame_base.wont_implement_method('plot')
+  hist = frame_base.wont_implement_method('hist')
```

Comment:
This was intentionally 'plot' for better summarization (e.g. to see that x% of methods that we won't implement are due to them being plotting). A traceback would point to the actual method used. But if we want to be more specific, that's OK too.